### PR TITLE
Added support for Hotjar Events API.

### PR DESCRIPTION
**Pull request by juliomalegria**
**What does this PR do?**

This PR adds `track` support for Hotjar integration script (Hotjar Events API).

**Are there breaking changes in this PR?**

No.

**Testing**

Testing completed successfully

```
  Hotjar Unit
    Constructing the integration
      ✓ should have the right settings
      ✓ should have the right settings with custom options
    #initialize
      ✓ should call this.ready() with valid options
      ✓ should prepare the needed globals
      ✓ should reject an invalid HJID
    after loading
      #identify
        ✓ should send id and traits
        ✓ should not send attributes when user is anonymous
      #track
        ✓ should send event and ignore properties
        ✓ should not send nameless event

HeadlessChrome 98.0.4758 (Mac OS X 10.15.7): Executed 9 of 9 SUCCESS (0.003 secs / 0.015 secs)
TOTAL: 9 SUCCESS
```

**Any background context you want to provide?**

Hotjar now has an Events API. It will be beneficial for Segment and Hotjar users to be able to make use of it seamlessly.

**Is there parity with the server-side/android/iOS integration components (if applicable)?**

No

**Does this require a new integration setting? If so, please explain how the new setting works**

No

**Links to helpful docs and other external resources**

https://help.hotjar.com/hc/en-us/articles/4405109971095-Events-API-Reference

### CODE REVIEW

File: integrations/hotjar/lib/index.js [hunk 1]
```diff
@@ -7,9 +7,7 @@ var is = require('is');
  * Expose `HotJar` integration
  */
 
-var Hotjar = (module.exports = integration('Hotjar')
-  .option('hjid', null)
-  .option('hjPlaceholderPolyfill', true));
```
Comment (kacper-hotjar) [inline integrations/hotjar/lib/index.js:12]: Why are we removing `hjPlaceholderPolyfill` option? :thinking:

File: integrations/hotjar/test/index.unit.test.js [hunk 2]
```diff
@@ -124,5 +115,32 @@ describe('Hotjar Unit', function() {
...
+      afterEach(function() {
+        analytics.reset();
+      });
+
+      it('should send event and ignore properties', function() {
+        analytics.stub(window, 'hj');
+        var event = 'the_event';
+        var properties = { a: 'a', b: 'b', c: [] };
+        analytics.track(event, properties);
+        analytics.called(window.hj, 'event', event);
```
Comment (kacper-hotjar) [inline integrations/hotjar/test/index.unit.test.js:134]: Will that test fail if we suddenly started sending properties (ideally it should imo)?

**Review by kacper-hotjar — APPROVED** [review]

File: integrations/hotjar/test/index.unit.test.js (hunk 2 above)
Comment (juliomalegria) [reply integrations/hotjar/test/index.unit.test.js:134]: I've added a `didNotCall` to assert that we're not including the properties for the time being.

File: integrations/hotjar/lib/index.js (hunk 1 above)
Comment (juliomalegria) [reply integrations/hotjar/lib/index.js:12]: We no longer support this option.

**Review by arielsilvestri — APPROVED** [review]
LGTM!

**Review by chrisradek — APPROVED** [review]
@juliomalegria Thank you so much for the update and apologies for the delay in reviewing. This looks great, we'll merge it right away!

**Comment by chrisradek** [comment]
Merging into feature branch - will create new PR from feature branch to trigger CI.

**Pull request merged**